### PR TITLE
API RH & PyTest & README & Crontab

### PR DESCRIPTION
**Pull request by RomTamam**
next time will be separated

### CODE REVIEW

**Comment by RomTamam** [comment]
Important to mention: API_FB PyTest removed due to work time of Facebook engineers and the PyTest was static.
Due to the lack of time, it been removed

**Review by Yarboa — COMMENTED** [review]
@RomTamam Please use git rm ./idea
It should not be added

It will not be added in the future

**Comment by RomTamam** [comment]
I think it came from my last pull, I didn't use this file

File: setup.sh [hunk 1]
```diff
@@ -14,7 +14,10 @@ pipenv sync
 pipenv run python manage.py makemigrations ultimatejobweb
 pipenv run python manage.py migrate
 
-
+#sudo dnf install crontabs
+#sudo systemctl start crond
+#python manage.py crontab add
```
Comment (Yarboa) [inline setup.sh]: Why it is all commented

Comment (RomTamam) [reply setup.sh]: mistake.. will be fixed now

**Pull request merged**